### DISCUSSION TITLE
Explaining the lack of Command Usage

### DISCUSSION THREAD

**Discussion by BC100Dev** [BeChris100/osintgram4j#18] [Announcements]
I believe that most of you still don't understand, how to use this tool. Can't blame you, since it is very much a different configuration set, along with its commands. The [original Project](https://github.com/Datalux/Osintgram) doesn't have such complexity. So, in this case, I should say this.

It is still in works. I already said that in the README, which is also cooperated as "in development". That should also mean that this project is nowhere completed.

Sure, I'm trying to respond to every issue that you guys make, but I can't update everyone on what each set of commands do. Plus, commands may change at any given monent, which means, why I reject in updating the `USAGE.md` file, until the project is completed, and reach the Testing stage.

So, what I am asking you guys is to stop making new issues. Yes, I got the info that it's still complex, and I would really suggest you guys to wait it out. I'll make sure to make a new announcement, once the commands are finished, along with ACTUALLY explaining every set of commands.

**Comment by SayanDas74** [comment]
So at this moment can we use it to some extent or not at all? I'm not able to login, typing user-manager login xyzuser does nothing. Please explain login and setting targets and commands

**Comment by BC100Dev** [comment]
> So at this moment can we use it to some extent or not at all? I'm not able to login, typing user-manager login xyzuser does nothing. Please explain login and setting targets and commands

That will be done, as soon as the Networking code is finished. Please be patient.

**Reply by DD-square** [reply]
is it fully furnished now? can i use it now?

**Reply by BC100Dev** [reply]
> is it fully furnished now? can i use it now?

I will make an announcement, along with a Release Candidate, once it is ready for use.
